src/Development/MASA.Utils.Development.Dapr/DaprProvider.cs

```diff
@@ -24,11 +24,11 @@ public List<DaprRuntimeOptions> GetDaprList(string appId)
                 }
             }
         };
-        _processUtils.Run(Const.DEFAULT_FILE_NAME, "list -o json", true, true);
         _processUtils.Exit += delegate
         {
             _logger?.LogInformation("{Name} process has exited", Const.DEFAULT_FILE_NAME);
         };
+        _processUtils.Run(Const.DEFAULT_FILE_NAME, "list -o json", true, true);
         string response = stringBuilder.ToString().Trim();
         List<DaprRuntimeOptions> daprList = new();
```

Comment:
The variable 'e' is declared but never used

Reply:
processed